fragment/fragment/src/main/java/androidx/fragment/app/FragmentLayoutInflaterFactory.java

```diff
@@ -130,6 +131,7 @@ public View onCreateView(@Nullable final View parent, @NonNull String name,
                         + "re-attached via the <fragment> tag: id=0x" + Integer.toHexString(id));
             }
         }
+        FragmentStrictMode.onFragmentTagUsage(fragment);
```

Comment:
I think we should make the call on line 74 before we check the ID and tag. The idea being that if you are using strict mode, we should throw this before we even check if you are using <fragment> tag correctly.

Reply:
I wanted to do that originally, but the problem there is that we don't have a `Fragment` instance yet. If there is a reliable way to traverse up the `FragmentManager` hierarchy without a concrete `Fragment` instance, that would also work, but I couldn't find one.

Reply:
There is a package private findFragmentManager that finds the FragmentManager associated with a given view, but I don't think that helps here much.